src/runtime/runtime-test.js

```diff
@@ -1945,6 +1945,21 @@ subscribe() method'
       expect(count).to.equal(1);
     });
 
+    it('should return the last OffersFlow', async () => {
+      let offersFlow;
+      sandbox.stub(OffersFlow.prototype, 'start').callsFake(function () {
+        offersFlow = this;
+        return new Promise(() => {});
+      });
+      // S how offers first in order to get an OffersFlow
+      runtime.showOffers();
+
+      const runtimeMock = sandbox.mock(runtime);
+      runtimeMock.expects('getLastOffersFlow').returns(offersFlow);
```

Comment:
I think this is creating a stub for this method, instead of testing the actual implementation